src/draco/io/splat_decoder.h

```diff
@@ -0,0 +1,44 @@
+// Copyright 2025 Patrick Trollmann.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+#ifndef DRACO_IO_SPLAT_DECODER_H_
+#define DRACO_IO_SPLAT_DECODER_H_
+
+#include <fstream>
+#include <vector>
+
+#include "draco/point_cloud/point_cloud.h"
+#include "draco/core/status.h"
+
+namespace draco {
+
+// Decodes a SPLAT file into draco::PointCloud.
```

Comment:
is there any official SPLAT file specification / definition that we could link to here?